Paladin.h

```diff
@@ -1,7 +1,21 @@
 #pragma once
+#include "Character.h"
 
-struct Paladin : Character
+struct Paladin : public Character
 {
 private:
     const std::string name;
-};
\ No newline at end of file
+
+public:
+    // Constructor
+    Paladin(std::string name_, int hp_, int armor_, int attackDamage_ = 10);
+
+    // Virtual Destructor
+    virtual ~Paladin() override;
+
+    // Overridden pure virtual function getName
+    virtual const std::string& getName() override;
+
+    // Overriden pure virtual function getStats
+    virtual std::string getStats() override;
+};
```

Comment:
```
 10) correct any unnecessary keywords related to access specifiers used in the UDTs.
```
`struct` is public by default
put public things at the top, private things at the bottom.
- [ ] revise